src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -88,14 +92,7 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
       setInputValue(event.target.value);
     }
 
-    const clickableProps = useClickableProps(
-      {
-        onClick: toggleEditMode,
-        id,
-        role: "button"
-      },
-      mergedRef
-    );
+    const toggleKeyboardEditMode = useKeyboardButtonPressedFunc(toggleEditMode);
```

Comment:
I couldn't use `useClickableProps` because it has a default role and this component has a more complex situation where the role changes, removing the default will be breaking so I just implemented clickable props here